frontend/packages/@depmap/interactive/src/components/Accordion.tsx

```diff
@@ -42,7 +42,9 @@ export default class Accordion extends React.Component<Props, State> {
     const { isOpen } = this.props;
 
     if (prevProps.isOpen !== isOpen) {
-      this.toggle();
+      if (isOpen !== undefined && prevProps.isOpen !== undefined) {
+        this.toggle();
+      }
```

Comment:
If I'm understanding correctly, I think this is very similar to something I tried to do at one point. @rcreasi  suggested a different hack that might be safer and worked quite well for me on the Data Page: https://github.com/broadinstitute/deprecated-depmap-repo/pull/717#discussion_r1586423492